src/components/Avatar/index.ts

```diff
@@ -1,4 +1,4 @@
 import Avatar from './Avatar';
 
-export * from './Avatar';
```

Comment:
@danrosenthal this should ensure we don't get the issue we experienced in web. There's no need to export * now that Avatar is a class component.